pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   enough_html_editor:
     git:
       url: https://github.com/linagora/enough_html_editor.git
-      ref: cnb_supported
+      ref: bugfix/signature_got_clipped
```

Comment:
We don't need change it. Only merged `bugfix/signature_got_clipped` branch in `enough_html_editor `